.github/workflows/biweekly.yml

```diff
@@ -0,0 +1,10 @@
+name: Biweekly
+
+on:
+  schedule:
+  - cron: "0 0 1,15 * *" # almost biweekly, twice a month
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci.yml
+    secrets: inherit
```

Comment:
Can we add a slack notification to #rqe_ci_discussions in case of failure?

Reply:
Done